src/naming_conventions.py

```diff
@@ -78,18 +83,19 @@ def convert_to_list(self, string_to_convert: str) -> list[str]:
         """
         strings_found = re.findall(self.regex, string_to_convert)
         if not strings_found:
-            raise Exception(
+            raise NamingConventionError(
                 f"{string_to_convert} is not of the expected naming convention."
             )
         else:
             for string_found in strings_found:
+                # TODO: this var is never used, what was the intention here?
```

Comment:
The intention here was to lowercase every character of every string found. Maybe my intention here was to change the value at the pointer for each index of the list `strings_found`. Not sure if it achieves this or if it's the best way if so.

Reply:
Python variables aren't pointers, they're references to values. This explains it pretty well: https://stackoverflow.com/questions/12080552/python-list-doesnt-reflect-variable-change/12080644#12080644

Reply:
I'll fix this method up as it is actually broken as LowerCase() is not a string method, you use .lower()